### DISCUSSION TITLE
Set/Get loop count of animated webp on ImageMagick6

### DISCUSSION THREAD

**Discussion by diego-choi** [ImageMagick/ImageMagick6#156] [General]
Hi, I found that it couldn't set/get loop count of animated webp on ImageMagick6. but I found this issue (https://github.com/ImageMagick/ImageMagick/discussions/2863) about this problem on ImageMagick7. 

I also found some commits for resolving this problem in IM7. so I think it is possible on IM7 now. 
* get loop count: https://github.com/ImageMagick/ImageMagick/commit/0e421edad2e6bc59894718fef2c93baadc0dfb11
* set loop count: https://github.com/ImageMagick/ImageMagick/commit/af1b1b66cada4e2832354095e15b0a4a55fdd502

Could you apply these improvements to IM6 too? or please let me know if I can be of any help for it. Thank you.

**Comment by dlemstra** [comment]
The code used in ImageMagick 6 is a lot different now because we added some extra features inside ImageMagick 7. Not sure when we will apply these changes to ImageMagick 6 or if we even can do that easily. We are open to pull requests for this.
